lib/lambda_deployment/configuration.rb

```diff
@@ -22,12 +43,28 @@ def alias_name
       tag
     end
 
+    def environment
+      @environment ||= calculate_environment_vars
+    end
+
     private
 
     def s3_key_name(file_name)
       basename = File.basename(file_name, '.*')
       extension = File.extname(file_name)
       "#{basename}-#{ENV.fetch('TAG', 'latest')}#{extension}"
     end
+
+    def calculate_environment_vars
+      Dir.glob('.env.*') do |filename|
+        File.open(filename).each do |line|
+          if (match = line.match(LINE_RE))
+            key, value = match.captures
+            @config_env[key] = value.strip
+          end
+        end
+      end
+      @config_env
```

Comment:
modifies `@config_env` and then gets cached in `@environment` ... kinda shady ... prefer side-effect free methods

Reply:
👍